cooltools/cli/coverage.py

```diff
@@ -80,15 +88,19 @@ def coverage(
 
     try:
         cis_cov, tot_cov = api.coverage.coverage(
-            clr, ignore_diags=ignore_diags, chunksize=chunksize, map=_map, store=store
+            clr, ignore_diags=ignore_diags, chunksize=chunksize, map=_map, store=store, clr_weight_name=clr_weight_name
         )
     finally:
         if nproc > 1:
             pool.close()
 
     coverage_table = clr.bins()[:][["chrom", "start", "end"]]
-    coverage_table["cis_raw_cov"] = cis_cov.astype(int)
-    coverage_table["tot_raw_cov"] = tot_cov.astype(int)
+    if clr_weight_name is None:
+        coverage_table["cis_raw_cov"] = cis_cov.astype(int)
+        coverage_table["tot_raw_cov"] = tot_cov.astype(int)
+    else:
+        coverage_table["cis_raw_cov"+str("_"+clr_weight_name)] = cis_cov.astype(float)
```

Comment:
we have to figure out something with the naming, as `cis_raw_cov_weigtht` does not seem readable either ...

maybe names like these would be better:
```python
["cis_cov_raw", "tot_cov_raw"],  # raw names when using counts
["cis_cov_weight", "tot_cov_weight"],  # balanced names ...
```

so, maybe we could replace `store_names=["cis_raw_cov", "tot_raw_cov"],` argument in `coverage` function with something like `store_prefixes=["cis_cov", "tot_cov"]` and then add `_raw`/`_clr_weight_name` suffix to combine raw/balanced scenarios together  - what do you guys think ?

Reply:
or maybe even simplier, like `store_prefix="cov"` and then just generate store name like so:
```python
if clr_weight_name is not None:
    store_names = [f"{store_prefix}_cis_{clr_weight_name}", f"{store_prefix}_tot_{clr_weight_name}" ]
else:
    store_names = [f"{store_prefix}_cis_raw", f"{store_prefix}_tot_raw" ]
```
or something like that